File: backend/slotify/spring/src/main/java/com/slotify/backend/spring/service/dtos/CreateServiceResponse.java
package com.slotify.backend.spring.service.dtos;

import lombok.Builder;
import lombok.Getter;
import lombok.ToString;

import java.util.UUID;

@Builder
@Getter
@ToString
public class CreateServiceResponse {

    private UUID serviceId;
    private String serviceName;
    private int minutesDuration;
    private int priceCent;
    private String description;
}

File: frontend/app/src/main/java/com/raj/slotify/dtos/service/CreateServiceResponse.kt
package com.raj.slotify.dtos.service

import com.google.gson.annotations.SerializedName
import java.util.UUID

data class CreateServiceResponse (

    @SerializedName("serviceId")
    val serviceId: UUID,

    @SerializedName("serviceName")
    val serviceName: String,

    @SerializedName("minutesDuration")
    val minutesDuration: Int,

    @SerializedName("priceCent")
    val priceCent: Int,

    @SerializedName("description")
    val description: String

)